tests/metrics/network/lib/network-test-common.bash

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2017 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+
+# This file contains functions that are shared among the networking
+# tests that are using nuttcp and iperf
+
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../../lib/test-common.bash"
+
+server_name="network-server"
+client_name="network-client"
+
+# This function will setup cor as a runtime, clean up the
+# environment and create a temporary file where networking
+# results will be stored
+function setup {
+	runtime_docker
+	kill_processes_before_start
+	result=$(mktemp)
+}
+
+# This function can be used in iperf and nuttcp networking tests
+function start_server {
+	$DOCKER_EXE run -d --name=${server_name} ${image} sh -c "${server_command}" > /dev/null
+	server_address=$($DOCKER_EXE inspect --format "{{.NetworkSettings.IPAddress}}" ${server_name})
+}
+
+# This function is mainly required in iperf and nuttcp networking tests
+function start_client {
+	$DOCKER_EXE run -ti --rm --name=${client_name} ${image} sh -c "${client_command}"
+}
+
+# This function is mainly required in pss memory networking tests
+function start_client_dettached {
```

Comment:
Typo: this should be `start_client_detached`.

Also, this function is almost identical to `start_client()` so you could instead do something like:

```
function start_client {
  local extra_args="$1"
  eval $DOCKER_EXE run $extra_args -ti --rm --name=${client_name} ${image} sh -c "${client_command}"
}

function start_client_detached() {
  start_client "-d"
}
```